### PR TITLE
correct metrics with scheme

### PR DESCRIPTION
**Pull request by Xiaohua-Shen**

### CODE REVIEW

**Comment by Xiaohua-Shen** [comment]
This PR depends on https://github.com/blueboxgroup/ursula-monitoring/pull/123

File: roles/ceph-monitor/tasks/monitoring.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@
 - name: ceph usage metrics
   sensu_metrics_check:
     name: ceph-usage-metrics
-    plugin: metrics-ceph.py
+    plugin: metrics-ceph-pool-capacity.rb
+    args='--scheme {{ monitoring.graphite.host_prefix }}'
```
Comment (paulczar) [inline roles/ceph-monitor/tasks/monitoring.yml]: does this need to be `args='--scheme {{ monitoring.graphite.host_prefix }}.ceph' ?   and is it a per host or per cluster metric ?

File: roles/nova-control/tasks/monitoring.yml [hunk 2]
```diff
@@ -64,14 +64,10 @@
...
   notify: restart sensu-client
 
-- name: nova quotas metrics
+- name: nova limit metrics (quota/usage)
   sensu_metrics_check:
-    name: compute-quotas-metrics
-    plugin: metrics-nova-project-quotas.py
+    name: compute-limit-metrics
+    plugin: metrics-openstack-project-limits.py
+    args='-S nova --scheme {{ monitoring.graphite.host_prefix }}'
```
Comment (paulczar) [inline roles/nova-control/tasks/monitoring.yml]: is this a per host or per cluster metric ?

**Comment by Xiaohua-Shen** [comment]
for ceph metrics:
1. already add prefix as  {{ monitoring.graphite.host_prefix }}.ceph
2. it's a cluster metrics, but it's existed on all 3 cpm nodes, and all active. should we let it run only on cpm1 and use cluster prefix? or still let it run on 3 cpm nodes with host prefix?

**Comment by nirajdp76** [comment]
ok to test

**Comment by epwn** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Pull request closed without merging**